src/daft-sql/src/planner.rs

```diff
@@ -468,6 +480,7 @@ impl SQLPlanner {
     }
 
     fn plan_compound_identifier(&self, idents: &[Ident]) -> SQLPlannerResult<Vec<ExprRef>> {
```

Comment:
I know that there are a lot of different things that a compound identifier could represent (duckdb examples: https://duckdb.org/docs/sql/data_types/struct.html#dot-notation-order-of-operations), which ones are we intending to support (in this PR as well as in the future)?

Reply:
right now we only support table access `tbl1.column`. struct access is currently only available through bracket notation `column["field"]`. Eventually, I think we should support struct access through dot notation though.